lib/stack_master/config.rb

```diff
@@ -74,7 +74,7 @@ def load_stacks(stacks)
         region = Utils.underscore_to_hyphen(region)
         stacks_for_region.each do |stack_name, attributes|
           stack_name = Utils.underscore_to_hyphen(stack_name)
-          stack_attributes = build_stack_defaults(region).deeper_merge(attributes).merge(
+          stack_attributes = build_stack_defaults(region).deeper_merge!(attributes).merge(
```

Comment:
The default s3 config couldn't be overridden on a per-stack basis. According to the [deep_merge documentation](https://github.com/danielsdeleo/deep_merge/blob/master/lib/deep_merge/deep_merge_hash.rb#L18) this is the expected result when using `deep_merge` (as opposed to `deep_merge!`).